LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -187,6 +187,305 @@ function groupListContent(groups) {
     return content;
 }
 
+// Automated Emails Section
+function emptyAlert(idElement) {
+    const elementId = document.getElementById(idElement).value;
+    if (elementId == "") {
+        alert("Please make a choice");
+    }
+}
+
+function createElement(typeOfElement, elementId, parentDiv) {
+    const newElement = document.createElement(typeOfElement);
+    newElement.setAttribute("id", elementId);
+    document.getElementById(parentDiv).appendChild(newElement);
+}
+
+function removeChild(removeChildId) {
+    const childId = document.getElementById(removeChildId);
+    if (childId) {
+        childId.removeChild(childId.firstElementChild);
+    }
+}
+
+function removeAllChildren(containerName) {
+    const container = document.getElementById(containerName);
+    while (container.firstChild) {
+        container.removeChild(container.firstChild);
+    }
+}
+
+function insertDOM(elementId, content) {
+    document.getElementById(elementId).innerHTML = content;
+}
+
+/**
+ * Email reminder dialog that will be triggered via the step popup
+ * @param stepID
+ */
+function addEmailReminderDialog(stepID){
+    $('.workflowStepInfo').css('display', 'none');
+    let workflowStep = null;
+    $.ajax({
+        type: 'GET',
+        data: {
+            CSRFToken: CSRFToken,
+
+        },
+        url: '../api/workflow/step/' + stepID,
+        async: false,
+        success: function (res) {
+            workflowStep = res
+        }
+    });
+
+    dialog.setTitle('Email Reminder');
+    let output = '<label for="edit_email_check">Automated Emails:</label> <input type="checkbox" id="edit_email_check" onclick="editEmailChecked()"><div id="edit_email_container"></div><br>';
+    dialog.setContent(output);
+    dialog.setSaveHandler(function() {
+
+        let seriesData = {
+            'Automate Email Group': $('#edit_email_check').prop('checked'),
+            'Date Selected': $('#edit_dates_selected').val(),
+            'Days Selected': $('#edit_dates_days').val()
+        }
+
+        $.ajax({
+            type: 'POST',
+            data: {
+                CSRFToken: CSRFToken,
+                seriesData: seriesData
+            },
+            url: '../api/workflow/stepdata/' + stepID,
+            success: function (res) {
+                if (res == 1) {
+                    //loadWorkflow(currentWorkflow);
+                    dialog.hide();
+                }
+                else {
```

Comment:
Are we following PSR-12? if so, that standard requires the } else { or } elseif { to be on the same line.

Reply:
Changed this and everything else I could find.